src/Microsoft.ML.TensorFlow/TensorFlowModel.cs

```diff
@@ -20,20 +19,20 @@ namespace Microsoft.ML.Transforms
     ///    </item>
     /// </list>
     /// </summary>
-    public class TensorFlowModelInfo
+    public sealed class TensorFlowModel
```

Comment:
I think possibly maybe there's some misunderstanding. Just to be explicit, I expect to see methods used to *query* the model to be *on the model*, that is, they will be just methods and properties of the model. Creatingtransformers, querying the schema, or *whatever*, will be here. The implication is that nearly everything in `TensorflowCatalog` will be moved out of there, and into here. (Except for loading the model, which of course must be on the model operations catalog.)